README.md

```diff
@@ -37,38 +37,51 @@ fastify.get('/__stats__', async function () {
   return this.stats()
 })
 
+// To group all methods of a route together:
+//   mergeMethods: true
+fastify.get('/mergeMethods', { config: { mergeMethods: true } }, function (request, reply) {
+  reply.send({ hello: 'world' })
+})
+
+fastify.post('/mergeMethods', { config: { mergeMethods: true } }, function (request, reply) {
```

Comment:
I think we could have a `mergeMethods` option at plugin-level. I don't think having this at route level make sense after all.